core/tests/protractor/collections.js

```diff
@@ -30,7 +31,6 @@ describe('Collections', function() {
       '.protractor-test-reload-collection-button')).first().click();
     general.acceptAlert();
     browser.waitForAngular();
-    admin.reloadAllExplorations();
```

Comment:
It looks like this (and the similar problem immediately below) is some kind of conflict with PR #3507. The code that appears to be deleted was not present before my commit. I followed the instructions at "Contributing code to Oppia", but I did notice my branch showed something like "1 commit ahead and 1 commit behind develop". Is there a name for this problem (a type of merge conflict) that I could look up to understand better?